app/jobs/state_file/send_still_processing_notice_job.rb

```diff
@@ -1,6 +1,7 @@
 module StateFile
   class SendStillProcessingNoticeJob < ApplicationJob
     def perform(submission)
+      return if submission.data_source.unsubscribed_from_sms?
```

Comment:
(dust) it would be much cleaner to use the enum convention instead of making a separate method
```suggestion
      return if submission.data_source.sms_notification_opt_in_no?
```